api/src/Entity/ContentNode/StoryboardSection.php

```diff
@@ -2,40 +2,80 @@
 
 namespace App\Entity\ContentNode;
 
+use ApiPlatform\Core\Annotation\ApiFilter;
 use ApiPlatform\Core\Annotation\ApiProperty;
 use ApiPlatform\Core\Annotation\ApiResource;
+use ApiPlatform\Core\Bridge\Doctrine\Orm\Filter\SearchFilter;
 use App\Entity\BaseEntity;
+use App\Entity\BelongsToCampInterface;
+use App\Entity\Camp;
 use App\Entity\SortableEntityInterface;
 use App\Entity\SortableEntityTrait;
+use App\InputFilter;
+use App\Repository\StoryboardSectionRepository;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Component\Serializer\Annotation\Groups;
 
 /**
- * @ORM\Entity
+ * @ORM\Entity(repositoryClass=StoryboardSectionRepository::class)
  * @ORM\Table(name="content_node_storyboard_section")
- * @ApiResource(routePrefix="/content_node")]
  */
-class StoryboardSection extends BaseEntity implements SortableEntityInterface {
+#[ApiResource(
+    routePrefix: '/content_node',
```

Comment:
Ist das das richtige prefix? Die section gehört ja zu einem contentnode, ist aber selber keiner

Reply:
Don't know. Ich hab StoryboardSection jetzt überall in den ContentNode namespace genommen, sowohl in der route als auch im Code. Dies aus der Überlegung, da die sections vollständig dem storyboard gehörten (anders als z.B. die material items).

Hast einen anderen Vorschlag?

Ich könnte mir vorstellen, dass man zukünftig je nach Komplexität den Namespace sogar noch stärker trennen will, z.B. für Marschzeittabelle:
/content_node/hikes
/content_node/hikes/1
/content_node/hikes/waypoints?hike=1
/content_node/hikes/segments?hike=1

Aber das schluckt dann wahrscheinlich der Router nicht.

Reply:
Ja ich finds eigentlich okay so wie dus gemacht hast, wollte nur nochmals anregen darüber nachzudenken. Dein Vorschlag gefällt mir auch sehr gut, wäre es für mich wert das mal auszuprobieren. Keine Ahnung wie schlau der Symfony Router ist.